src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -459,6 +413,49 @@ internal void ApplyConflictResolutionEdits(IInlineRenameReplacementInfo conflict
                 }
             }
 
+            private static async Task<IEnumerable<TextChange>> GetTextChangesFromTextDifferencingServiceAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken = default(CancellationToken))
+            {
+                try
+                {
+                    using (Logger.LogBlock(FunctionId.Workspace_Document_GetTextChanges, newDocument.Name, cancellationToken))
+                    {
+                        if (oldDocument == newDocument)
+                        {
+                            // no changes
+                            return SpecializedCollections.EmptyEnumerable<TextChange>();
+                        }
+
+                        if (newDocument.Id != oldDocument.Id)
+                        {
+                            throw new ArgumentException(WorkspacesResources.DocumentVersionIsDifferent);
+                        }
+
+                        SourceText oldText = await oldDocument.GetTextAsync(cancellationToken).ConfigureAwait(false);
+                        SourceText newText = await newDocument.GetTextAsync(cancellationToken).ConfigureAwait(false);
+
+                        if (oldText == newText)
+                        {
+                            return SpecializedCollections.EmptyEnumerable<TextChange>();
+                        }
+
+                        var textChanges = newText.GetTextChanges(oldText).ToList();
```

Comment:
But can we just update GetTextChanges to not give up? It seems odd to have an API that doesn't do what you might want but looks like it does for maximum confusion.